src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/Logging/SuppressionEngineTests.cs

```diff
@@ -51,14 +51,14 @@ public void SuppressionEngineCanParseInputSuppressionFile()
         [Fact]
         public void SuppressionEngineThrowsIfFileDoesNotExist()
         {
-            Assert.Throws<FileNotFoundException>(() => new SuppressionEngine(suppressionFile: "AFileThatDoesNotExist.xml"));
+            Assert.Throws<FileNotFoundException>(() => new SuppressionEngine("AFileThatDoesNotExist.xml"));
         }
 
         [Fact]
         public void SuppressionEngineDoesNotThrowOnEmptyFile()
         {
-            SuppressionEngine _ = new(suppressionFile: string.Empty);
-            _ = new SuppressionEngine(suppressionFile: "      ");
+            SuppressionEngine _ = new(suppressionsFile: string.Empty);
+            _ = new SuppressionEngine("      ");
```

Comment:
NIT: not a big deal, but why not also keep a named parameter here to match the above?